packages/frontend/src/components/common/TwoFactorDisableBanner.js

```diff
@@ -0,0 +1,170 @@
+            height: 25px;
+        }
+     
+    }
+    
+    .content {
+        margin-right: 20px;
+    }
+
+    .title {
+        font-weight: 700;
+        color: #DC1F25;
+    }
+
+    .desc {
+        margin-top: 10px;
+        color: #DC1F25;
+    }
+
+    && {
+        button {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            margin: 0 !important;
+            height: 40px;
+            padding: 8px 16px;
+            white-space: nowrap;
+            border-radius: 50px;
+            font-weight: 600;
+            .lock-icon {
+                margin: 0;
+                margin-right: 10px;
+            }
+            :hover {
+                > svg {
+                    path {
+                        stroke: #E5484D;
+                    }
+                }
+            }
+        }
+
+        @media (max-width: 767px) {
+            button {
+                width: 100%;
+                margin-top: 25px !important;
+            }
+        }
+    }
+
+    @media (max-width: 767px) {
+        flex-direction: column;
+    }
+`;
+
+export default function TwoFactorDisableBanner() {
+    const [accounts, setAccounts] = useState([]);
+    const [showDisable2FAModal, setShowDisable2FAModal] = useState(false);
+
+    const account = useSelector(selectAccountSlice);
+    const loadedAccounts = useMemo(() => Object.keys(account.accounts ?? {}), [account.accounts]);
+
+    const showModal = () => setShowDisable2FAModal(true);
+    const hideModal = () => setShowDisable2FAModal(false);
+
+    useEffect(() => {
+        const update2faAccounts = async () => {
+            const accounts = await wallet.keyStore.getAccounts(NETWORK_ID);
+            const getAccountWithAccessKeysAndType = async (accountId) => {
+                const keyType = await wallet.getAccountKeyType(accountId);
+                return { accountId, keyType };
+            };
+            const accountsKeyTypes = await Promise.all(
+                accounts.map(getAccountWithAccessKeysAndType)
+            );
+    
+            setAccounts(accountsKeyTypes.reduce(((acc, { accountId, keyType }) => keyType === WalletClass.KEY_TYPES.MULTISIG ? [...acc, accountId] : acc), []));
+        };
+        if (loadedAccounts.length > 0 && accounts.sort() !== loadedAccounts.sort()) {
+            update2faAccounts();
+        }
+    }, [showDisable2FAModal, loadedAccounts.length]);
+
+    const accountsCount = accounts.length;
+    if (accounts.length === 0) {
+        return null;
+    }
+
+    return (
+        <Container className='banner-container'>
+            <div className='alert-container'>
+                <AlertTriangleIcon color={'#DC1F25'} />
+            </div>
+            <div className='content'>
+                <h4 className='title'>
+                    <Translate id='twoFactorDisbleBanner.title' data={{ accountsCount, context: accountsCount === 1 ? 'Account Needs' : 'Accounts Need'}} />
```

Comment:
`context: accountsCount === 1 ? 'Account Needs' : 'Accounts Need'}}`

The above won't work with other languages. You may have to split the title string into several strings to handle singular vs plural. You can render the number of accounts directly in the code since it won't change across languages.

Reply:
Thanks for the feedback, follow up PR has been created [here](https://github.com/near/near-wallet/pull/2942)